packages/config/jest.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  moduleNameMapper: {
+    '^src/(.*)$': '<rootDir>/src/$1',
```

Comment:
Needed to import `defineConfig` in the `prisma.config.ts` files located under `packages/config/src/__tests__/fixtures`.